### ISSUE TITLE
G6 Android Pie Active Sensor Session Won't Connect

### ISSUE THREAD

**Issue by davebrothers** [NightscoutFoundation/xDrip#863]
| Who | What |
| --- | --- |
|Phone | Pixel 3 (Android 9) |
| xDrip+ Build | Current nightly (20190413), set to get latest nightly on update; have also tried latest stable with same behavior |
| CGM | Dexcom G6 (also paired with a Tandem t:slim insulin pump) |
| Primary Issue | xDrip+ will not connect to my transmitter with an active sensor session in progress |

I am new to G6 (2 weeks) and have not used xDrip[+] before.

Because my phone supports it, I am using the official (Play Store-distributed) G6 app. I am running both G6 app and xDrip+ on my Pixel 3 at the same time (is this my problem?). I would like to run them side-by-side if possible to decide which I like better. Hesitant to use xDrip+ only because of these connection issues; don't want to waste a sensor.

G6 app works fine. I do not see my transmitter in my phone's connected Bluetooth devices list (should I? I have read many people describing seeing it present as normal). The transmitter does appear in previously connected devices list. In the xDrip+ classic status page, it shows by ID with the message `Dexcom6L Authed`

After starting a sensor session in G6 app, I cannot also connect using xDrip+. If I enter the transmitter information, I get the `Now start your sensor or check settings` message. In G5/G6 Status page, just shows scanning perpetually. 

Attempted several times to also "start sensor" using the xDrip+ source setup wizard, but System Status did the same perpetual scanning status with the Start G6 Sensor action in queue on the Status page and `G6 State isn't currently known. Next connection will update this.` on home screen.

Have also tried (with no changes): 
- Disabling the Trust Auto-Connect as described in #609
- Toggling OB1 off (aggressively re-enabled when restarting collector)
- Toggling Native Algorithm off

**Comment by lmiekus** [comment]
Uninstall official dexcom app from your phone. This is causing your problem. They can't run at the same time

**Comment by ruiannie** [comment]
I have the same issue. If Dexcom is installed, it works, Xdrip does not work if Dexcom is uninstalled. Samsung Galaxy Note 8.

**Comment by davebrothers** [comment]
@lmiekus thanks for your response earlier. I had just started a sensor session on G6. I will try xDrip+ sometime soon, but as your suggestion makes sense for my problem, I'm going to close this.

I'm not entirely sure about @ruiannie 's comment; it sounds like the opposite of my issue with respect to the presence of Dexcom G6 app. If this is incorrect, we can certainly reopen it.

**Comment by ruiannie** [comment]
I've uninstalled the Dexcom app and experience the same issue.

**Comment by goemic** [comment]
I experience the same error message on:
Google Pixel 3
Android 10
latest xdrip nightly build

Unfortunately xdrip never receives data :-/

**Comment by SLYDoggie** [comment]
Everything was working fine on my Samsung Note 10+. I even updated to the new version a couple of weeks ago. I updated my phone to the latest Verizon Android 10 and now I'm getting the G6 State isn't currently known. I didn't have the problem with the Verizon Note 10+ software update until I had to go to a new sensor. It worked for a very short time and but I've been getting this message for over 10 hours now. I have started/stopped Bluetooth, restarted the xDrip+ collector, and have also restarted my phone. None of those got it working again.
